geometry_msgs/PoseStamped goal
bool ignoreOrientation
---
bool success
